### PR TITLE
OCPQE-16304: Add cert-rotation tests in ROSA

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-rosa-sts-f28-cert-rotation

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-aws-rosa-sts-f28-cert-rotation

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 1]
```diff
@@ -617,6 +617,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts
+- as: aws-rosa-sts-f28-cert-rotation
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:620]: frequency should be the last field in the job name, right?

**Comment by xingxingxia** [comment]
ROSA STS needs unlock https://github.com/openshift/release/blob/8d0c987/ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh#L131 otherwise some operators will be abnormal. Steps are in google doc https://url.corp.redhat.com/fe70bfb (internal). For more info, @wangke19 , please chime in.

**Pull request closed without merging**